packages/realm/tsconfig.react-native.json

```diff
@@ -0,0 +1,25 @@
+{
+  "extends": "@react-native/typescript-config/tsconfig.json",
+  "compilerOptions": {
+    "composite": true,
+    "module": "node16",
+    "moduleResolution": "node16",
+    "outDir": "dist/platform/react-native",
+    "rootDir": "src/platform/react-native",
+    "sourceMap": true,
+    "resolveJsonModule": true,
+    "noEmit": false,
```

Comment:
This is the default value right? (P.S. the node ts config file did not explicitly specify it)

Reply:
The react-native config that we extend sets this to `true`, so we need to override that here.